full-stack-tests/backend/src/integration/AzuriteTest.ts

```diff
@@ -54,7 +54,7 @@ export namespace AzuriteTest {
       return blobClient.exists();
     };
 
-    export const setWriteLockToExpired = async (container: CloudSqlite.CloudContainer, blockName: string) => {
+    export const resetWriteLockExpireTime = async (container: CloudSqlite.CloudContainer, blockName: string, numOfMin: number) => {
```

Comment:
maybe this should be called 'subtractFromCurrentWriteLockExpiryTime' ?

Reply:
Resolved in [commit 8a0fdb5818d3144a873d3ae42a56c9a99db05473](https://github.com/iTwin/itwinjs-core/pull/7000/commits/8a0fdb5818d3144a873d3ae42a56c9a99db05473)